whylog/tests/tests_log_reader/test_read.py

```diff
@@ -29,6 +32,19 @@ def setUpClass(cls):
             )
         )  # yapf: disable
 
+    def test_getting_line_by_offset(self):
```

Comment:
a file looking like this:
```
12
45
78
```
can be used to demonstrate methods from `ReadUtils` class. Either show how they work in tests, or document them - this is not a trivial code and it should be documented somehow. I'd prefer tests over a docstring explanation.